job-service-db/src/main/resources/procedures/getJobs.sql

```diff
@@ -206,7 +206,7 @@ BEGIN
                 FOR UPDATE;
 
             -- Process outstanding job updates
-            PERFORM internal_update_job_progress(in_partition_id, jobId);
+            PERFORM internal_update_job_progress(in_partition_id, job);
```

Comment:
Actually I don't get this.  Isn't `jobId` correct, as `job` is an array?

Reply:
the jobId wasn't being fed. job as an array was unnecessary. updateJobProgress wasn't throwing an error for a null value as it's only checking the type and we were passing a varchar[] which is considered as valid